### ISSUE TITLE
Logging framework?

### ISSUE THREAD

**Issue by danwatt** [schemaspy/schemaspy#190]
In a recent PR it was suggested I use SLF4j to log an error. SLF4j was already on the classpath, but, never used. There are 28 classes that import `java.util.logging.Logger`.

Can I get direction on which logging framework we want to use going forward?

I am about to make a pass to clean up 15 or so more Sonar warnings where we are not properly logging exceptions, but want to be sure to use the correct logging framework.

**Comment by npetzall** [comment]
If this is up for a vote I say slf4j since this might get used as a library #157

**Comment by wkasa** [comment]
+1 for slf4j

**Comment by npetzall** [comment]
If slf4j is used getLogger(Methodhandles.lookup().lookupclass())

**Comment by rafalkasa** [comment]
+1 also for slf4j

**Comment by npetzall** [comment]
I would say we are migrating to slf4j.

In that sense I would suggest we add this to a section in readme.md could be our first contribution guidelines.

Use slf4j, if it class your working with has any other logging replace it with slf4j.
Use jUnit for testing.
Use assertj for assertions.

Try to use clean code. Nobody's perfect but one could aim for clean code.

**Comment by npetzall** [comment]
@danwatt if your not on gitter, get on gitter :)

**Comment by danwatt** [comment]
Does that mean we get rid of a lot of the `if (level is enabled) { }` wrappers?

Ex:
``` java
 if (fineEnabled) {
                logger.fine("Excluding " + clazz + " " + name +
                        ": embedded $ implies illegal name");
            }
```

to
``` java
logger.trace("Excluding {} {} : embedded $ implies illegal name",clazz,name);
```

**Comment by rafalkasa** [comment]
I will create this  basic contribution assumptions 😄

**Comment by npetzall** [comment]
I would suggest we skip all form of level checking we aren't building high performance jet engines so better to have less noise in the code.

**Comment by danwatt** [comment]
Solved in #191
